src/linked_list.py

```diff
@@ -0,0 +1,127 @@
+"""Link list implementation."""
+import sys
+
+
+class Node(object):
+    """Define node object."""
+
+    def __init__(self, value, next_up):
+        """Instantiate an object with a given value."""
+        self.value = value
+        self.next = next_up
+
+
+class Linked_List(object):
+        """Define the Link_List class structure"""
+    def __init__(self, optional_values=[]):
```

Comment:
This isn't the best idea. Since the parameter should be optional you should probably start it out with a value of `None`